core/tee/tee_svc_storage.c

```diff
@@ -657,6 +657,56 @@ TEE_Result tee_svc_storage_obj_del(uint32_t obj)
 	return TEE_SUCCESS;
 }
 
+TEE_Result tee_svc_storage_obj_del1(uint32_t obj)
```

Comment:
There is a lot of duplicated code from `tee_svc_storage_obj_del()` here, please use a common helper function or something instead.